### DISCUSSION TITLE
Front-end Cypress Tests for Scaladoc - Update or delete it ?

### DISCUSSION THREAD

**Discussion by Dedelweiss** [scala/scala3#17401] [Contributing Questions] [answered]
Hello everyone,

during a meeting with Chris Kipp, we noticed that Scaladoc cypress front-end tests were present for the Scaladoc, [scaladoc/e2e](https://github.com/lampepfl/dotty/tree/main/scaladoc/e2e).
But we also noticed that the last changes to these tests were made 3 years ago. Scaladoc has evolved a lot since then.

The only way to be sure is to try these tests. So I use the Readme and I do the following.
But when I run the command `npm run cypress:open`, I get the following page:
<img width="1000" alt="Screenshot 2023-05-03 at 17 00 38" src="https://user-images.githubusercontent.com/44496264/235956828-7d739549-4077-4c58-8b50-ba291ec21c11.png">
I guess this is not the expected page.  So I try to change the starting URL to 'http://localhost:8080/testcases/index.html' where a page does exist.

 However, when testing I notice that none of them get through. 
<img width="1000" alt="Screenshot 2023-05-03 at 17 00 28" src="https://user-images.githubusercontent.com/44496264/235957102-4dc261cb-861e-432d-821c-c8a434a93dea.png">
 
I guess this is normal (if I didn't make a mistake during the process) considering the different updates of Scaladoc since 3 years.
So, here is the subject I would like to address, do we want to keep these tests and therefore update them or do we delete them?

**Comment by ckipp01** [comment]
@romanowski I see you originally worked on this. Do you have any thoughts on this?

**Reply by ckipp01** [reply]
@Florian3k maybe you have some thoughts here?

**Reply by Florian3k** [reply, marked as answer]
I don't know much about these tests.
It seems that they were only testing the filter bar. I think it all comes down to the question of whether updating them will save us more time in the long run, and I'm not so sure about that.

**Reply by ckipp01** [reply]
Yes, I totally agree. I've gone ahead and just created https://github.com/lampepfl/dotty/pull/17960 to remove these.
